packages/web/src/pages/pay-and-earn-page/components/PayoutWalletCard.tsx

```diff
@@ -30,6 +33,16 @@ export const PayoutWalletCard = () => {
     setIsOpen(true)
   }, [setIsOpen])
 
+  const { value: payoutWallet } = useAsync(async () => {
+    if (user?.spl_usdc_payout_wallet) {
+      const owner = await getUSDCAssociatedTokenAccountOwner(
+        user.spl_usdc_payout_wallet
+      )
+      return owner.toString()
```

Comment:
super nit but I like doing `toBase58()` which should only exist on public keys in case this type changes (since everything has a `toString()` this won't break if the return type changes)

Reply:
nice ty